spec/external_data_sets_two_electric_boogaloo_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+RSpec.describe Facility do
+  before(:each) do
+    co_dmv = DmvDataService.new.co_dmv_office_locations
+    @facility = FacilityObjects.new
+  end
+
+  it 'can initialize' do
+    expect(@facility).to be_an_instance_of(FacilityObjects)
+  end
+  #require 'pry';binding.pry
+   it 'can find dmv data using co dmv api' do
+    co_dmv = DmvDataService.new.co_dmv_office_locations
+    new_york_facilities = DmvDataService.new.ny_dmv_office_locations
+    missouri_facilities = DmvDataService.new.mo_dmv_office_locations
+    @facility = FacilityObjects.new
+    fac_co = @facility.create_facilities(co_dmv)
+    fac_co.each do |fac|
+     expect(fac).to be_a(Facility)
+     #expect(fac.phone).to be_a(?)
+    end
+     #require 'pry';binding.pry
+   end
+
+end
```

Comment:
This test looks good for the basic functionality you're looking for. We'd also want to test that we can create Facility objects for any of the DMV data sets that we have. We'd ALSO want to test any helpe rmethods that you created.